public-api/src/test/java/org/pmiops/workbench/cdr/dao/SurveyModuleDaoTest.java

```diff
@@ -0,0 +1,62 @@
+package org.pmiops.workbench.cdr.dao;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.List;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.cdr.model.DbSurveyModule;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
```

Comment:
I would keep the annotation
@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
This cleans your database between each test

Reply:
Added this to test class. Thanks.